### DISCUSSION TITLE
Persistent releaseType Identifier (or Other)

### DISCUSSION THREAD

**Discussion by flyingfishflash** [mooltiverse/nyx#203] [Q&A] [answered]
I'm finding that with _releaseTypes/\<NAME\>/identifiers_ the configured identifiers are only included in the computed version if there is a significant commit in the release scope. This makes sense.

Is it possible to have the computed version always include a configured identifier in the computer version?

Example: If I'd prefer to have the branch name included in the BUILD position of the computed version for the _internal_ release type, regardless if there are any significant commits in the release scope. Is there a way to do that with the current configuration options? Might be missing something obvious but I couldn't figure it out.

Thanks,
-Greg

**Comment by flelli** [comment, marked as answer]
Generally speaking, no new version is generated unless there are some significant commits. In case there are none, the latest version is retrieved from the tag history and used as the current version. This is by design.

What you can do is create a custom commit message convention to make Nyx detect more commits as significant. Nonetheless you still need new commits for that.

In case you have no new commits at all Nyx is somehow out of scope. In that case I sugges to just modify the version number by adding custom identifiers, maybe using regular expressions.
